data/data/baremetal/masters/outputs.tf

```diff
@@ -0,0 +1,13 @@
+output "control_plane_ips" {
+  value = <<EOT
+    var.master_count > 2 ? [
+      data.ironic_introspection.openshift-master-introspection.interfaces.0.ip,
+      data.ironic_introspection.openshift-master-introspection.interfaces.1.ip,
+      data.ironic_introspection.openshift-master-introspection.interfaces.2.ip ] :
+        var.master_count > 1 ? [
+          data.ironic_introspection.openshift-master-introspection.interfaces.0.ip,
+          data.ironic_introspection.openshift-master-introspection.interfaces.1.ip ] :
+            var.master_count > 0 ? [
+              data.ironic_introspection.openshift-master-introspection.interfaces.0.ip ] : 0
+EOT
```

Comment:
Does wildcard work here?
```
value = data.ironic_introspection.openshift-master-introspection.interfaces.*.ip
```

Example: https://github.com/openshift/installer/blob/05e59bb8082491eff09ba594c73f42dc7bf27be5/data/data/gcp/cluster/master/outputs.tf#L2

Reply:
The wildcard does work, and I have changed it for the case where all 3 IP's are needed.